Makefile

```diff
@@ -46,30 +46,6 @@ prettier-check:
 prettier:
 	prettier --write --plugin-search-dir=. src
 
-browser-test:
-	docker compose -f local.builder.yml run --rm browser-test
-
-browser-test-staging:
-	docker compose -f local.builder.yml run --rm browser-test-staging
-
-browser-test-direct:
-	node tests/functional/suites/noindex.js
-
-browser-test-direct-all:
-	BROWSER=firefox node tests/functional/suites/noindex.js
-	#BROWSER=chromium node tests/functional/suites/noindex.js
-	BROWSER=webkit node tests/functional/suites/noindex.js
-
-# Set BROWSER to change the browser. e.g. BROWSER=chromium make browser-test-headful
-browser-test-headful:
-	DEBUG=yes node tests/functional/suites/noindex.js
-
-browser-test-headful-staging:
-	DEBUG=yes node tests/functional/suites/noindex.js --envfile .env.staging
-
-browser-test-debug:
-	DEBUG=yes node debug tests/functional/suites/noindex.js
-
```

Comment:
Why are these removed?

Reply:
We're not using them anymore. Browser tests run outside of Docker now, and these refer to files we've already deleted.

Reply:
Also deleted the `plugins` folder I made for Netlify that we ended up not using.